src/__tests__/companiesSlice.test.js

```diff
@@ -0,0 +1,30 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { fetchCompanies } from '../redux/companies/companiesSlice';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const mock = new MockAdapter(axios);
+
+describe('companiesSlice', () => {
+  afterEach(() => {
+    mock.reset();
+  });
+
+  it('should fetch companies successfully', async () => {
+    const responseData = [
+      'AAIC',
+      'BBDI',
+    ];
```

Comment:
- [ ] When I ran all your tests, some of them were failing. Kindly address them 

![image](https://github.com/leloufadel/Financial-Metrics-App/assets/98453979/880139a2-8ccd-473c-b547-0b3ca751e3f4)